niryo_one_commander/scripts/arm_moveit_commander.py

```diff
@@ -100,6 +100,10 @@ def check_trajectory(self, plan):
             #rospy.loginfo("Sign change counter : " + str(sign_change_counter))
         return True
     
+    
```

Comment:
the "plan" here is not the same as the "plan" executed in the self.arm.execute(self.next_plan, wait=False), so it might be confusing in the future. Maybe just use self.next_plan = plan and 
+            if cmd_type == CommandType.EXECUTE_TRAJ:
+                status, message = self.set_plan_and_execute(cmd.plan) --> use cmd.plan.trajectory here ?

Reply:
makes more sense yes